web-animations-1/Overview.bs

```diff
@@ -3363,6 +3364,9 @@ The <a>effect value</a> of a single property referenced by a <a>keyframe
 effect</a> as one of its <a lt="target property">target properties</a>, for a
 given <var>iteration progress</var>, <var ignore=''>current iteration</var> and
 <var>underlying value</var> is calculated as follows.
+Note that this procedure is only performed on a <a>keyframe effect</a> having
+an <a>effect target</a> for which computed property values can be calculated,
+as otherwise case all propereties are <a>not animatable</a>.
```

Comment:
As above, I'd probably avoid invoking the "not animatable" language since that's really an animation type and I think it is confusing to use that concept here. It's probably better just to say this procedure is not run on that case, or, better still, add a step that early returns if the the effect target is in a context where we can't calculate property values.

Reply:
removed